### PR TITLE
Added support for Inbound Security Rule for Network Virtual Appliance

### PR DESCRIPTION
**Pull request by ashutmi**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Accounts/Accounts/Utilities/CommandMappings.json [hunk 1]
```diff
@@ -4598,6 +4598,7 @@
       "New-AzVHubRoute": {},
       "New-AzVHubRouteTable": {},
       "New-AzVirtualApplianceAdditionalNicProperty": {},
+      "New-AzVirtualApplianceInboundSecurityPropertyRules": {},
```
Comment (YanaXu) [inline src/Accounts/Accounts/Utilities/CommandMappings.json:4601]: don't update this file.

**Comment by YanaXu** [comment]
Hi @ashutmi , the code freeze date of the next Major release is 05/13. Please update the PR today or it goes to the next release after the major release.

File: src/Accounts/Accounts/Utilities/CommandMappings.json (hunk 1 above)
Comment (ashutmi) [reply src/Accounts/Accounts/Utilities/CommandMappings.json]: Removed the changes added to this file.

File: src/Network/Network/NetworkVirtualAppliance/InboundSecurityRule/UpdateVirtualApplianceInboundSecurityRule.cs [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+                        WriteVerbose(String.Format(Properties.Resources.CreatingLongRunningOperationMessage, this.ResourceGroupName, this.Name));
+                        WriteObject(this.CreateOrUpdateInboundSecurityRule(this.ResourceGroupName, this.VirtualApplianceName, this.Name, inboundSecurityRuleParameters));
+                    }
+                );
+        }
+
+    }
+
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualApplianceInboundSecurityPropertyRules",
```
[the remark this answers is not in the document]
Comment (ashutmi) [reply src/Network/Network/NetworkVirtualAppliance/InboundSecurityRule/UpdateVirtualApplianceInboundSecurityRule.cs]: Created 2 files and separated the changes

**Review by YanaXu — APPROVED** [review]

**Pull request merged**